doc/nrf/doc_build_process.rst

```diff
@@ -0,0 +1,78 @@
+.. _doc_build_process:
+
+Documentation build process
+###########################
+Whenever the main branch of the `sdk-nrf`_ repository is updated or a pull request is made, a new version of the documentation is built and published. The following section describes the steps involved with building and publishing the documentation.
+
+Building with Sphinx
+********************
+The |NCS| documentation uses the Sphinx framework to build the docsets for HTML.
+Sphinx primarily uses three files to work:
+
+* A makefile that runs the :code:`sphinx-build` command in the correct folder. This folder should contain the next two files.
```

Comment:
note: we don't use makefiles, but CMake. Still, it's not a Sphinx requirement so I think this entry is not needed